### PR TITLE
Fix error handling.

### PR DESCRIPTION
**Pull request by captainbeardo**
Sometimes the error isn't a hash, but just an array of messages. For some reason.

### CODE REVIEW

File: lib/conduit/reactor/parsers/base.rb [hunk 1]
```diff
@@ -74,10 +74,15 @@ def normalized_errors
 
         response_errors.map do |attribute, error|
           # error is array of messages, explode into unique errors.
```
Comment (mgates) [inline lib/conduit/reactor/parsers/base.rb]: This comment doesn't seem true anymore

File: lib/conduit/reactor/parsers/base.rb [hunk 2]
```diff
@@ -74,10 +74,15 @@ def normalized_errors
...
-        end.flatten
+          if error.nil?
+            # if the error_messages is nil, the attribute is just a message
+            Conduit::Error.new(message: attribute)
+          else
+            # error is array of messages, explode into unique errors.
+            error.map do |message|
+              Conduit::Error.new(message: message, attribute: attribute)
+            end
+          end        end.flatten
```
Comment (mgates) [inline lib/conduit/reactor/parsers/base.rb]: This line is two lines.

**Comment by mgates** [comment]
Is this supposed to be against master?

**Comment by captainbeardo** [comment]
Yes, all the conduit drivers only have master, no staging.

**Comment by pedromartinez** [comment]
👍

**Comment by mgates** [comment]
Oh right. 👍 .

**Pull request merged**